projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -153,25 +153,25 @@ import { createSelector, createFeatureSelector } from '@ngrx/store';
 export const featureKey = 'feature';
 
 export interface FeatureState {
-  counter: number;
+counter: number;
 }
 
 export interface AppState {
-  feature: FeatureState;
+feature: FeatureState;
 }
```

Comment:
The `AppState` interface can be removed, because it's not used anymore:

```diff
-export interface AppState {
-  feature: FeatureState;
-}
```

Reply:
Removed AppState interface here and the removed the feature slice section below.